dash_labs/plugins/pages.py

```diff
@@ -272,13 +305,16 @@ def update(pathname, search):
                     layout = html.H1("404")
 
             if callable(layout):
-                print("Calling...")
-                print(query_parameters)
-                return layout(**query_parameters)
+                return (
+                    layout(**path_variables, **query_parameters)
```

Comment:
should we raise an exception if the same variable is used as a `path_variable` and a `query_parameter`?

Reply:
or just a warning?  I assume in the example above the `query_parameters` would replace any duplicates in `path_variables`?

Reply:
Yeah, that's what the code would do. Good idea re warning 👍🏼

Reply:
Actually, do you think warnings would get noisy (and not useful) in productions since end users could potentially enter the conflicts? If this warning is for the developer, then is it really necessary?  It's probably not too hard to debug by looking at the URL.